syntax/atom.rs

```diff
@@ -20,6 +20,18 @@ pub enum Atom {
     F64,
     CxxString,
     RustString,
+    CInt,
+    CDouble,
+    CFloat,
```

Comment:
These are just identical to `f64` and `f32` on every platform right? If so, I would prefer not to add another name for them.

Reply:
Done